specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/2021-03-01/connectedClusters.json

```diff
@@ -0,0 +1,698 @@
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the connected cluster. The type 'SystemAssigned, includes a system created identity. The type 'None' means no identity is assigned to the connected cluster.",
+          "enum": [
+            "None",
+            "SystemAssigned"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Identity for the connected cluster."
+    },
+    "ConnectedClusterProperties": {
+      "type": "object",
+      "required": [
+        "agentPublicKeyCertificate"
+      ],
+      "properties": {
+        "agentPublicKeyCertificate": {
+          "type": "string",
+          "description": "Base64 encoded public certificate used by the agent to do the initial handshake to the backend services in Azure."
+        },
+        "kubernetesVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Kubernetes version of the connected cluster resource"
+        },
+        "totalNodeCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "Number of nodes present in the connected cluster resource"
+        },
+        "totalCoreCount": {
+          "readOnly": true,
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of CPU cores present in the connected cluster resource"
+        },
+        "agentVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Version of the agent running on the connected cluster resource"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ConnectedClusterProvisioningState",
+          "description": "Provisioning state of the connected cluster resource."
+        },
+        "distribution": {
+          "type": "string",
+          "description": "The Kubernetes distribution running on this connected cluster."
+        },
+        "infrastructure": {
+          "type": "string",
+          "description": "The infrastructure on which the Kubernetes cluster represented by this connected cluster is running on."
+        },
+        "offering": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Connected cluster offering"
+        },
+        "managedIdentityCertificateExpirationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Expiration time of the managed identity certificate"
+        },
+        "lastConnectivityTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "Time representing the last instance when heart beat was received from the cluster"
+        },
+        "connectivityStatus": {
```

Comment:
>connectivityStatus [](start = 9, length = 18)

Should this be readonly?

Reply:
I think it should be readonly...@Brijesh can you confirm?

Reply:
I believe it should be readonly..Brijesh can you confirm?

Reply:
marked it as readonly